.env.example

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_BASE_URL= # main url on production
+# nothing
```

Comment:
Why is this removed?

Reply:
karena `base url`-nya udah depend ke `next-sitemap.js`-nya mas, jadi saya pikir udah nggak ke pakai lagi

Reply:
@kaliwa Oke, tapi please keep this ya. Sebisa mungkin config seperti ini jangan di-expose di config file, taruh di env aja. Bisa dipake juga buat misalnya ada fitur baru lain yang butuh.

Reply:
beres mas. Tapi saya masih belum ketemu solusi kenapa waktu tes production, `env` di `next-sitemap.js` valuenya nggak kebaca. Mungkin mas @resir014 tahu solusinya?

Reply:
saya sudah ketemu solusinya, mungkin dari mas @resir014 ada yang better solution?

from [reference](https://github.com/iamvishnusankar/next-sitemap/issues/38)

Reply:
Oh oke. For now gak apa-apa ditaruh di `next-sitemap.js`, tapi env-nya jangan dihapus di file `.env.example`.